src/main/java/maria/TaskManager.java

```diff
@@ -0,0 +1,48 @@
+package maria;
+
+import maria.command.CommandExecutor;
+import maria.task.TaskList;
+import maria.util.StorageConverter;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+/**
+ * Acts as the interface for all the tasks related logic in the program.
```

Comment:
Very minor, but using another term instead of "interface" could be better as it might be confused with the Java Interface.